dotnet/Dockerfile

```
@@ -7,3 +7,7 @@ RUN wget -q https://packages.microsoft.com/config/ubuntu/19.04/packages-microsof
     add-apt-repository universe && \
     apt-get update && apt-get -y -o APT::Install-Suggests="true" install dotnet-sdk-3.1 && \
     rm -rf /var/lib/apt/lists/*;
+RUN sudo apt-get update \
+    && sudo apt-get install -y \
+        fsharp \
+        mono-complete
```

Please include these packages in the RUN instruction above:

```suggestion
    apt-get update && \
    apt-get -y -o APT::Install-Suggests="true" install dotnet-sdk-3.1 && \
    apt-get install -y fsharp mono-complete && \
    rm -rf /var/lib/apt/lists/*
```